nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -174,6 +174,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - `services.mastodon` gained a tootctl wrapped named `mastodon-tootctl` similar to `nextcloud-occ` which can be executed from any user and switches to the configured mastodon user with sudo and sources the environment variables.
 
+- `services.nginx` gained a `defaultListen` option at server-level with support for PROXY protocol listeners, also `proxyProtocol` is now exposed in `services.nginx.virtualHosts.<name>.listen` option. It is now possible to run PROXY listeners and non-PROXY listeners at a server-level, see [#213510](https://github.com/NixOS/nixpkgs/pull/213510/) for more details.
```

Comment:
Can you please copy the relevant part from your comment into here?

Reply:
"the relevant part" -> what do you mean exactly?

Reply:
I am not sure but it would be nice if people while reading the changelog don't need to trace this PR and filter what is relevant. :)

Reply:
Got it

Reply:
good point